### ISSUE TITLE
add 'interlaced', 'anamorphic', h.264 level+profile+refFrames and 'optimized for streaming' properties

### ISSUE THREAD

**Issue by OttoKerner** [ukdtom/plex2csv.bundle#83]
Since Plex2CSV is often used to bring a video collection into shape and optimize it for Plex, these properties are somewhat important:

1. interlaced video - leads to transcoding on a number of devices
2. anamorphic video - some older Rokus don't like it
3. h.264 level - anything above 4.1 or 4.2 leads to transcoding on a big number of clients 
4. h.264 profile 
5. h.264 number of reference frames - some devices have rather strict rules for # of ref frames (Roku, once again)
6. optimized for streaming - is a property exclusive to mp4/m4v but an important one. This alone could trigger a transcode with some clients.

"3. + 4." could be combined within the 'codec' field, so it would read `h.264 - high 10 - L5.1` with "high" being the profile which is another important part of identifying the codec [and the '10' after it would indicate 10 bit color depth, which is another big reason for transcoding])

some examples:

mp4, optimized for streaming, anamorphic, 1 ref frame
```
<Part accessible="1" exists="1" id="745549" key="/library/parts/745549/file.mp4" duration="6627280" file="xxxxxx.mp4" size="1266436346" container="mp4" has64bitOffsets="0" indexes="sd" optimizedForStreaming="1" videoProfile="main">
<Stream id="976419" streamType="1" default="1" codec="h264" index="0" bitrate="1077" language="English" languageCode="eng" anamorphic="1" bitDepth="8" cabac="0" chromaSubsampling="4:2:0" codecID="avc1" duration="6627280" frameRate="25.000" frameRateMode="cfr" hasScalingMatrix="0" height="404" level="41" pixelAspectRatio="404:405" pixelFormat="yuv420p" profile="main" refFrames="1" scanType="progressive" streamIdentifier="1" width="720"/>
<Stream id="976420" streamType="2" selected="1" default="1" codec="ac3" index="1" channels="6" bitrate="448" language="Deutsch" languageCode="ger" audioChannelLayout="5.1(side)" bitDepth="16" bitrateMode="cbr" codecID="ac-3" dialogNorm="-27" duration="6627264" samplingRate="48000" streamIdentifier="2"/>
</Part>
```
MPEG2, interlaced, anamorphic
```
<Part accessible="1" exists="1" id="28034" key="/library/parts/28034/file.mkv" duration="6627280" file="xxxxxx.mkv" size="4394475759" container="mkv" indexes="sd" videoProfile="main">
<Stream id="181274" streamType="1" codec="mpeg2video" index="0" bitrate="4751" language="English" languageCode="eng" anamorphic="1" bitDepth="8" chromaSubsampling="4:2:0" codecID="V_MPEG2" colorRange="tv" duration="6627280" frameRate="25.000" frameRateMode="cfr" height="576" pixelAspectRatio="64:45" pixelFormat="yuv420p" profile="main" scanType="interlaced" width="720"/>
<Stream id="181275" streamType="2" selected="1" codec="ac3" index="1" channels="6" bitrate="448" language="Deutsch" languageCode="ger" audioChannelLayout="5.1(side)" bitDepth="16" bitrateMode="cbr" codecID="A_AC3" dialogNorm="-27" duration="6627280" samplingRate="48000" title="Surround 5.1"/>
</Part>
```

Anime with h.264 level 5.0, 10bit color depth and 16 reference frames
```
<Part accessible="1" exists="1" id="104358" key="/library/parts/104358/file.mkv" duration="1422940" file="xxxxxx.mkv" size="439592878" audioProfile="lc" container="mkv" indexes="sd" videoProfile="high 10">
<Stream id="190308" streamType="1" default="1" codec="h264" index="0" bitrate="2471" language="日本語" languageCode="jpn" bitDepth="10" cabac="1" chromaSubsampling="4:2:0" codecID="V_MPEG4/ISO/AVC" duration="1422923" frameRate="23.976" frameRateMode="cfr" hasScalingMatrix="0" height="720" level="50" pixelFormat="yuv420p10" profile="high 10" refFrames="16" scanType="progressive" title="xxxxxx" width="1280"/>
<Stream id="190309" streamType="2" selected="1" default="1" codec="aac" index="1" channels="2" language="日本語" languageCode="jpn" audioChannelLayout="stereo" codecID="A_AAC" duration="1422940" profile="lc" samplingRate="48000" title="AAC 320 Kbps"/>
```

**Comment by ukdtom** [comment]
Oh boy....

I let the wolf to the sheep's here ;-)

**Comment by OttoKerner** [comment]
I propose a new detail level, called "insane". ;-)

**Comment by ukdtom** [comment]
it looks like 'pixelaspectRatio' should contain a number, then a colon and a second number. But it contains only one number. These two numbers are the pixel width and pixel height. But these are already listed in other columns. 
For this to be a bit more useful, you should solve the fraction and present a single number, runded to 2 decimal places. 
e.g. width 1920 / height 1078 = pixel aspect ratio `1.78 `

**Comment by ukdtom** [comment]
And disregard above.....

**Comment by ukdtom** [comment]
Fixed in V0.0.4.0
